src/Util.php

```diff
@@ -22,6 +22,24 @@ public static function convertToObject($class, array $values)
         return $object;
     }
 
+    /**
+     * Converts values into an Invoiced object; forces `id` to placeholder `-1`.
+     *
+     * @param object $class
+     * @param array  $values
+     *
+     * @return object
+     */
+    public static function convertPreviewToObject($class, array $values)
```

Comment:
If you change the subscription preview endpoint to return a JSON object then you can drop this function.

Reply:
This function is dropped